lib/src/release.dart

```diff
@@ -120,7 +120,8 @@ class Release {
   void _formatCode(String srcPath, bool usingGit) {
     final output = <String>[];
 
-    'dartfmt -w $srcPath'.forEach((line) => output.add(line), stderr: print);
+    'dart format --summary none $srcPath'
+        .forEach((line) => output.add(line), stderr: print);
```

Comment:
Are the dart format options new as the current 2.12 options are: 

dart format
Missing paths to code to format.

Usage: dart format [options...] <files or directories...>
-h, --help                     Print this usage information.
-v, --verbose                  Show all options and flags with --help.
-o, --output                   Set where to write formatted output.

          [json]               Print code and selection as JSON.
          [none]               Discard output.
          [show]               Print code to terminal.
          [write] (default)    Overwrite formatted files on disk.

    --set-exit-if-changed      Return exit code 1 if there are any formatting changes.
    --fix                      Apply all style fixes.
-l, --line-length              Wrap lines longer than this.
                               (defaults to "80")

Run "dart help" to see global options.

I would normally use:

dart format --fix $srcPath

Reply:
No it is not new you need to run:
```
dart format -v
```
Then it prints all the options for the formater.
```
sboyanov@linux-ialq:~> dart format -v
Can only use --verbose with --help.

Usage: dart format [options...] <files or directories...>
-h, --help                             Print this usage information.
-v, --verbose                          Show all options and flags with --help.

Output options:
-o, --output                           Set where to write formatted output.

          [json]                       Print code and selection as JSON.
          [none]                       Discard output.
          [show]                       Print code to terminal.
          [write] (default)            Overwrite formatted files on disk.

    --show                             Set which filenames to print.

          [all]                        All visited files and directories.
          [changed] (default)          Only the names of files whose formatting is changed.
          [none]                       No file names or directories.

    --summary                          Show the specified summary after formatting.

          [line] (default)             Single-line summary.
          [none]                       No summary.
          [profile]                    How long it took for format each file.

    --set-exit-if-changed              Return exit code 1 if there are any formatting changes.

Non-whitespace fixes (off by default):
    --fix                              Apply all style fixes.
    --fix-doc-comments                 Use triple slash for documentation comments.
    --fix-function-typedefs            Use new syntax for function type typedefs.
    --fix-named-default-separator      Use "=" as the separator before named parameter default values.
    --fix-optional-const               Remove "const" keyword inside constant context.
    --fix-optional-new                 Remove "new" keyword.
    --fix-single-cascade-statements    Remove unnecessary single cascades from expression statements.

Other options:
-l, --line-length                      Wrap lines longer than this.
                                       (defaults to "80")
-i, --indent                           Add this many spaces of leading indentation.
                                       (defaults to "0")
    --follow-links                     Follow links to files and directories.
                                       If unset, links will be ignored.
    --version                          Show dart_style version.

Options when formatting from stdin:
    --selection                        Track selection (given as "start:length") through formatting.
    --stdin-name                       Use this path in error messages when input is read from stdin.
                                       (defaults to "stdin")

Run "dart help" to see global options.
```

Reply:
I did `--summury none` as the tool relies on the output of the formatted to so it could `git add` the changed files. I think you could do just adding all files at the end of the command.